### ISSUE TITLE
Adjust props for drawTextCell

### ISSUE THREAD

**Issue by ALMaclaine** [glideapps/glide-data-grid#630]
```
export function drawTextCell(
    args: Pick<BaseDrawArgs, "rect" | "ctx" | "theme">,
    data: string,
    contentAlign?: BaseGridCell["contentAlign"],
    allowWrapping?: boolean,
    hyperWrapping?: boolean
)
```

Here are the props for drawTextCell. It's exported to use, but it's somewhat difficult to type args.

BaseDrawArgs isn't exported, but I tried using it anyways and I can't get my args to match. Can you create a type def for args and export it?

Thank you

**Comment by jassmith** [comment]
Fixed in 5.2.2

**Comment by ALMaclaine** [comment]
@jassmith will toy be releasing this soon?
